src/spender/paths/spender@expense_policies@states.yaml

```diff
@@ -0,0 +1,60 @@
+get:
+  tags:
+    - Expense Policies
+  summary: Get policy states of an Expense
+  description: |
+    Get policy states of an Expense
+  operationId: expense_policy_states
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: ../../components/schemas/expense_policies.yaml#/expense_id_in
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            additionalProperties: False
+            properties:
+              count:
+                $ref: ../../components/schemas/count.yaml
+              offset:
+                $ref: ../../components/schemas/offset.yaml
+              data:
+                type: array
+                items:
+                  $ref: ../../components/schemas/expense_policies.yaml#/expense_states_out
+    '400':
```

Comment:
just confirm from any live api(s) all these error codes are being used with the same semantics

Reply:
For most of them, it is not even used (I think it should have been used in those too)
But wherever it is used, it is used with the same semantics.